README.md

```diff
@@ -1,17 +1,301 @@
+# ...
+
+Successfully built 2452b4b1fce1
+Successfully tagged my-app:my-tag
 ```
+
+The message `DETECTING WITH MANUALLY-PROVIDED GROUP` indicates that the buildpack was chosen by the user, rather than
+by the automated detection process.
+
+The `--buildpack` parameter can be
+- a path to a directory
+- a path to a `.tgz` file
+- a URL to a `.tgz` file, or
+- the ID of a buildpack located in a builder
+
+### Building Explained
+
+![](docs/build-diagram.png)
+
+To create an app image, `build` executes one or more buildpacks against the app's source code.
+Each buildpack inspects the source code and provides relevant dependencies. An image is then generated
+from the app's source code and these dependencies.
+
+Buildpacks are compatible with one or more [stacks](#managing-stacks). A stack designates a **build image**
+and a **run image**. During the build process, a stack's build image becomes the environment in which buildpacks are
+executed, and its run image becomes the base for the final app image. For more information on working with stacks, see
+the [Managing Stacks](#managing-stacks) section.
+
+Buildpacks can be bundled together with a specific stack's build image, resulting in a
+[builder](#working-with-builders-using-create-builder) image (note the "er" ending). Builders provide the most
+convenient way to distribute buildpacks for a given stack. For more information on working with builders, see the
+[Working with Builders Using `create-builder`](#working-with-builders-using-create-builder) section.
+
+## Updating App Images Using `rebase`
+
+The `pack rebase` command allows app developers to rapidly update an app image when its stack's run image has changed.
+By using image layer rebasing, this command avoids the need to fully rebuild the app.
+
+```bash
+$ pack rebase <image-name>
+```
+
+### Example: Rebasing an App Image
+
+Consider an app image `my-app:my-tag` that was originally built using the default builder. That builder's stack has a
+run image called `pack/run`. Running the following will update the base of `my-app:my-tag` with the latest version of
+`pack/run`.
+
+```bash
+$ pack rebase my-app:my-tag
+```
+
+Like [`build`](#building-app-images-using-build), `rebase` has a `--publish` flag that can be
+used to publish the updated app image to a registry.
+
+### Rebasing Explained
+
+![](docs/rebase-diagram.png)
+
+At its core, image rebasing is a simple process. By inspecting an app image, `rebase` can determine whether or not a
+newer version of the app's base image exists (either locally or in a registry). If so, `rebase` updates the app image's
+layer metadata to reference the newer base image version.
+
+## Working with Builders Using `create-builder`
+
+`pack create-builder` enables buildpack authors and platform operators to bundle a collection of buildpacks into a
+single image for distribution and use with a specified stack.
+
+```bash
+$ pack create-builder --builder-config <path-to-builder-toml>
```

Comment:
pack create-builder my-builder-image:tag --builder-config <path-to-builder-toml>

Reply:
@ssisil Good catch!